docs/en/admins/08_FeedUpdates.md

```diff
@@ -1,19 +1,56 @@
 # Setting Up Automatic Feed Updating
 
-FreshRSS updating is controlled by a script located at `./app/actualize_script.php`. Knowing this, we can create a Cron job to launch the update script.
+FreshRSS updating is controlled by a script located at `./app/actualize_script.php`. Knowing this, we can trigger the script periodically to have up-to-date feeds.
 
-**Note:** the update script will not fetch feeds more often than every twenty minutes, so there's no sense in setting the Cron job to run anymore often than that.
+**Note:** the update script will not fetch feeds more often than every twenty minutes, thus triggering on a shorter period is counterproductive.
```

Comment:
This is not exactly accurate: feeds can be set down to 15 minutes individually. Furthermore, if the cron job is set exactly to the same frequency than the feeds, and since it takes time to refresh all the feeds, the next refresh might not include those feeds, refreshing them only every second cron job.

Reply:
But let's keep for another pull request, this it was not the point of this one

Reply:
As written it was already inaccurate; as updated more so. Maybe something like this:

```suggestion
**Note:** the update script won't update any particular feed more often than once every twenty minutes, so it doesn't make sense to trigger the cron job much more frequently than that.
```